src/comp/comp.py

```diff
@@ -25,47 +30,67 @@ def __repr__(self):
 # whose name starts with 'D':
 print("Starts with D:")
 a = []
+for i in humans:
+    if i.name.startswith('D'):
+        a.append(i.name)
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
 b = []
+for i in humans:
+    if i.name.endswith('e'):
+        b.append(i.name)
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
+# c = [i for i in humans if re.match(r'[C-G, c-g]', str(i))]
 c = []
+for i in humans:
+    if re.match(r'[C-G, c-g]', i.name):
+        c.append(i.name)
```

Comment:
Hey I used regex too for this! Nice. Don't forget the `in` keyword for lists. 
Example:
`c = [human.name for human in humans if human.name[0] in 'CDEFG']`